From: Mark Dufour <m.dufour@zarafa.com>
Date: Sat, 24 Dec 2016 09:23:09 +0100
Subject: adjust search path for php-config script on php7.0

The macro autoconf/php.m4 is relying on a non existing path in Debian
for php-config in PHP7.
---
 autoconf/php.m4 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/autoconf/php.m4 b/autoconf/php.m4
index 3c2f3a7..bf45408 100644
--- a/autoconf/php.m4
+++ b/autoconf/php.m4
@@ -25,10 +25,10 @@ AC_DEFUN([PHP_WITH_PHP_CONFIG],[
 	# find path in existing paths
 	if test -d "/etc/php.d"; then
 		PHP_SYSCONF_DIR="/etc/php.d"
-	elif test -d "/etc/php7/conf.d"; then
-		PHP_SYSCONF_DIR="/etc/php7/conf.d"
-	elif test -d "/etc/php7/apache2/conf.d"; then
-		PHP_SYSCONF_DIR="/etc/php7/apache2/conf.d"
+	elif test -d "/etc/php/7.0/conf.d"; then
+		PHP_SYSCONF_DIR="/etc/php/7.0/conf.d"
+	elif test -d "/etc/php/7.0/apache2/conf.d"; then
+		PHP_SYSCONF_DIR="/etc/php/7.0/apache2/conf.d"
 	elif test -d "/etc/php5/conf.d"; then
 		PHP_SYSCONF_DIR="/etc/php5/conf.d"
 	elif test -d "/etc/php5/apache2/conf.d"; then
